src/include/daos/checksum.h

```diff
@@ -173,12 +190,17 @@ daos_csummer_init(struct daos_csummer **obj, struct csum_ft *ft,
  *			for it.
  * @param type		Type of the checksum algorithm that will be used
  * @param chunk_bytes	Chunksize, typically from the container configuration
+ * @param srv_verify	Whether to verify checksum on the server on update
+ * @param dedup		Whether deduplication is enabled
+ * @param dedup_verify	Whether to memcmp data on the server for deduplication
+ * @param dedup_bytes	Deduplication size threshold
  *
  * @return		0 for success, or an error code
  */
 int
 daos_csummer_type_init(struct daos_csummer **obj, enum DAOS_CSUM_TYPE type,
-		       size_t chunk_bytes, bool srv_verify);
+		       size_t chunk_bytes, bool srv_verify, bool dedup,
+		       bool dedup_verify, size_t dedup_bytes);
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?